src/components/map/MapDirective.js

```diff
@@ -63,22 +64,75 @@ goog.require('ga_permalink');
 
             // Update permalink based on view states.
             var updatePermalink = function() {
-              var center = view.getCenter();
-              var zoom = view.getZoom();
-              // when the directive is instantiated the view may not
-              // be defined yet.
-              if (center && zoom !== undefined) {
-                var x = center[1].toFixed(2);
-                var y = center[0].toFixed(2);
-                gaPermalink.updateParams({X: x, Y: y, zoom: zoom});
+              // only update the permalink in 2d mode
+              if (!scope.ol3d || !scope.ol3d.getEnabled()) {
+                // remove 3d params
+                gaPermalink.deleteParam('lon');
+                gaPermalink.deleteParam('lat');
+                gaPermalink.deleteParam('elevation');
+                gaPermalink.deleteParam('heading');
+                gaPermalink.deleteParam('pitch');
+                var center = view.getCenter();
+                var zoom = view.getZoom();
+                // when the directive is instantiated the view may not
+                // be defined yet.
+                if (center && zoom !== undefined) {
+                  var x = center[1].toFixed(2);
+                  var y = center[0].toFixed(2);
+                  gaPermalink.updateParams({X: x, Y: y, zoom: zoom});
+                }
               }
             };
             view.on('propertychange', gaDebounce.debounce(updatePermalink,
                 1000, false));
-            updatePermalink();
```

Comment:
This is a regression. 2D parameters arent' in PL anymore after page load and without user interaction.

But I think it's a good thing. And in 3D it's the same now.